upgrade_tests/test_scenarios/test_capsule.py

```diff
@@ -137,25 +137,21 @@ def test_post_user_scenario_capsule_sync(self):
             2. The repos/rpms from satellite should be synced to satellite
 
          """
-        cap_host = os.environ.get(
-            'RHEV_CAP_HOST',
-            os.environ.get('CAPSULE_HOSTNAME')
-        )
         cap_data = hammer.hammer('capsule list')
-        cap_id = hammer.get_attribute_value(cap_data, cap_host, 'id')
-        cap_info = {'id': cap_id, 'name': cap_host}
+        cap_id = hammer.get_attribute_value(cap_data, self.cap_host, 'id')
         org_data = hammer.hammer('organization list')
         org_name = hammer.get_attribute_value(
             org_data, int(self.org_id), 'name')
-        print hammer.sync_capsule_content(cap_info, async=False)
+        print hammer.hammer(
```

Comment:
The main change is here to fix the issue,

We will now sync environments which were already selected in the capsule. Where before fix we were selecting all the available environments, adding them to satellite and syncing.

Reply:
ok.. just to clear.. sync_capsule_content fn actually sync all environments to capsule which is not a ideal solution. we should sync only the one being added to capsule.